### PR TITLE
docusaurus files and gitaction workflows added(LATEST)

### PR DESCRIPTION
**Pull request by davidmitesh**
Documentation website using Docusaurus added to the /docs. Also, GitHub action workflow is added which gets triggered if the PR is merged to main.

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: Deploy to GitHub Pages
+
+on:
+  push:
+    branches:
+      - main
+      - 'docs/**'
+      - '.github/workflows/deploy.yml'
```
Comment (Glitch18) [inline .github/workflows/deploy.yml:8]: I think we only need to list the develop branch here.

**Review by Glitch18 — COMMENTED** [review]
Tested out the docusaurus website locally. Overall looks good to me.
Let's make the commented change in the Github Action Workflow. The deploy job should run for this PR after that and we can test out the github action.

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (davidmitesh) [reply .github/workflows/deploy.yml:8]: But Guy told me that the update should only trigger when the PR is merged to the main.

Comment (Glitch18) [reply .github/workflows/deploy.yml:8]: I see, okay. But what about the other two items? do we need them as target branches?

Comment (davidmitesh) [reply .github/workflows/deploy.yml:8]: no those are the target subdirectories and files, to which only if update is made then only the update takes place.

Comment (Glitch18) [reply .github/workflows/deploy.yml:8]: I think you should be able to remove the other two and only keep main as the target branch for this workflow.

Comment (Glitch18) [reply .github/workflows/deploy.yml:8]: Right, for that we would need to use the `paths` event under `push`. Something like:

```
on:
    push:
        paths:
          - 'docs/**'
```

Under `branches` it should only be main

Comment (Glitch18) [reply .github/workflows/deploy.yml:8]: See: https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#using-filters-to-target-specific-paths-for-pull-request-or-push-events

Comment (davidmitesh) [reply .github/workflows/deploy.yml]: sorry my bad, wanted to do that, mistakely wrote that under the branches. corrected now

**Review by Glitch18 — APPROVED** [review]

File: docs/blog/authors.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+maceo:
```
Comment (GuyMcComb) [inline docs/blog/authors.yml:1]: Please change this to your name.

File: docs/blog/2022-06-24-junopi/index.mdx [hunk 3]
```diff
@@ -0,0 +1,127 @@
+---
+# slug: junopi
+# title: Running Juno from your Raspberry Pi
+title : Blog template 2 (WIP)
+# authors: [depablos, maceo]
+# tags: [juno, rpi, raspberry, pi, deploy]
+---
+
+<!-- Juno is a node which aims to help **decentralise StarkNet, a prominent Ethereum Layer 2**.
+
```
Comment (GuyMcComb) [inline docs/blog/2022-06-24-junopi/index.mdx:10]: All of this will not appear right? Will it just say WIP?

File: docs/package.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
+{
+  "name": "juno-docs",
```
Comment (GuyMcComb) [inline docs/package.json:2]: This should change

**Comment by GuyMcComb** [comment]
The Juno images are still present, please remove them.

File: docs/blog/authors.yml (hunk 2 above)
Comment (davidmitesh) [reply docs/blog/authors.yml]: done

File: docs/blog/2022-06-24-junopi/index.mdx (hunk 3 above)
Comment (davidmitesh) [reply docs/blog/2022-06-24-junopi/index.mdx:10]: yeah

File: docs/package.json (hunk 4 above)
Comment (davidmitesh) [reply docs/package.json]: done

**Comment by davidmitesh** [comment]
> 

All images from juno are commented I guess. Did you find any one left?

**Comment by GuyMcComb** [comment]
> > 
> 
> All images from juno are commented I guess. Did you find any one left?

[docs/static/img/juno_rounded.png](https://github.com/NethermindEth/warp/pull/640/files#diff-f81e46760c856ac409e3f4d33cbce0be6575483ee9da9ff34f096067bc3cf581)

**Review by GuyMcComb — APPROVED** [review]

**Pull request merged**